src/utils/datetime.js

```diff
@@ -110,3 +110,18 @@ export function millisecondsAsTimeOfDay(milliseconds, format = 'h:mm a') {
   }
   return moment.utc(milliseconds).format(format);
 }
+
+/**
+ * formatDisplayDate
+ * @param  {Number} timestamp integer hammertime
+ * @param  {Object} timePrefs object containing timezoneAware Boolean and timezoneName String
+ *
+ * @return {String}           formatted timezoneAware date string
+ */
+export function formatDisplayDate(timestamp, timePrefs) {
+  if (!timestamp) {
+    return '';
```

Comment:
Can you look at the other functions for a model of how to handle incorrect inputs rather than returning an empty string? @jh-bate and I looked at this when combining the two functions from the branches that had the (now) `millisecondsAsTimeOfDay` functionality. My experience with this tricky time stuff is we don't want these functions to silently fail; let's be consistent about throwing informative errors if the inputs are not as expected. If you look at a method like `localNoonBeforeTimestamp` we're being specified about *not* getting a`Date` (because of the historical issues with the `Date` constructor you and I discussed yesterday that have made me eternally wary). I think it'd be great to align (and I can update code too) re: what else to error on, like complete lack of input and whether we'll even accept a string rather than an integer hammertime.